src/units.ts

```diff
@@ -97,6 +98,110 @@ export function convertValue(value: number, unit: string) {
   }
 }
 
+export function convertUnits(value: number, unit: string, outputUnit: string) {
+  // We want to switch between specific units
+  switch (unit) {
+    case 'Grams': {
+      switch (outputUnit) {
+        case 'Ounces': {
+          return value * 0.035274;
+        }
+        case 'Pounds': {
+          return value * 0.002204;
+        }
+        case 'Milligrams': {
+          return value * 1000;
+        }
+        case 'Kilograms': {
+          return value * 0.001;
+        }
+        default: {
+          return value;
+        }
+      }
+    }
+    case 'Kilograms': {
+      switch (outputUnit) {
+        case 'Ounces': {
+          return value * 35.274;
+        }
+        case 'Pounds': {
+          return value * 2.20462;
+        }
+        case 'Milligrams': {
+          return value * 1000000;
+        }
+        case 'Grams': {
+          return value * 1000;
+        }
+        default: {
+          return value;
+        }
+      }
+    }
+    case 'Milligrams': {
+      switch (outputUnit) {
+        case 'Ounces': {
+          return value * 0.035274;
+        }
+        case 'Pounds': {
+          return value * 2.20462e-6;
```

Comment:
Not sure if we may run into very large number representations at this time. If/when the time comes, we may need to look into something like [big.js](https://www.npmjs.com/package/big.js?activeTab=readme)